parsely/src/androidTest/java/com/parsely/parselyandroid/FunctionalTests.kt

```diff
@@ -0,0 +1,121 @@
+package com.parsely.parselyandroid
+
+import android.app.Activity
+import androidx.test.core.app.ActivityScenario
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.fasterxml.jackson.core.type.TypeReference
+import com.fasterxml.jackson.databind.ObjectMapper
+import java.io.File
```

Comment:
Minor (🔍): For some reason, when I run reformat on this file it moved those imports at the end, I am not sure if you have a different imports configuration on your side, if that's the default or not, but maybe we can align on that so that we don't get a diff between us. 🤔

Reply:
I don't know, maybe: I use "Default IDE" in this project. I think it's okay, as it's sorted alphabetically.

Reply:
Yea, I think the default IDE is per the scheme, which applies to all projects, not a specific project (🤷). Maybe we should compare our default and update them on our imports, either you or me, both to have us both on the same page...

Reply:
Oh I see now! I think about introducing `detekt` to this project soon, so probably we won't need to sync the settings - the `detekt` default config will do that for us. WDYT?

Reply:
Yea, let's forget about this until `Detekt` is added to this project, deal! 👍